index.js

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env node
+//require('../')()
+
+const findLinks = require('./utils/findLinks')
+const linksFunctions = require('./utils/validateAndStatusLinks')
+const fs = require('fs')
+const path = require('path')
+const chalk = require('chalk')
+
+const readDirectory=(uri, comandLine4, comandLine5)=>{
+    let direc = fs.readdirSync(uri)
```

Comment:
Sugiero definir mejor la variable
```suggestion
    let Folder = fs.readdirSync(uri)
```